js/src/dropdown.js

```diff
@@ -241,17 +241,26 @@ const Dropdown = (() => {
       return placement
     }
 
+    _getOffset(data) {
+      data.offsets = $.extend({}, data.offsets, this._config.offset(data.offsets) || {})
+      return data
+    }
+
     _detectNavbar() {
       return $(this._element).closest('.navbar').length > 0
     }
 
     _getPopperConfig() {
+      const offsetConf = {}
+      if (typeof this._config.offset === 'function') {
+        offsetConf.fn = (data) => this._getOffset(data)
+      } else {
```

Comment:
I was thinking maybe we should remove `_getOffset` method because that's not something we will use outside of this and remplaced it by : 

```javascript
offsetConf.fn = (data) => {
     data.offsets = $.extend({}, data.offsets, this._config.offset(data.offsets) || {})
     return data
}

Can you give it a try ?

Reply:
Remove this method but in my opinion this is less readable. On the other hand, the method is not needed, because obviously not planned to manage offseting inside boostrap